coala_quickstart/green_mode/green_mode.py

```diff
@@ -0,0 +1,537 @@
+import fnmatch
+import itertools
+import os
+import yaml
+from copy import deepcopy
+from pathlib import Path
+
+from coala_quickstart.generation.Utilities import (
+    split_by_language, peek, contained_in)
+from coala_quickstart.generation.SettingsClass import (
+    SettingTypes)
+from coala_quickstart.green_mode.QuickstartBear import (
+    QuickstartBear)
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.processes.Processing import (
+    get_file_dict, yield_ignore_ranges)
+from coalib.settings.Section import Section
+
+
+settings_key = 'green_mode_infinite_value_settings'
+
+
+def get_contents(project_data):
+    """
+    Reads a YAML file and returns the data.
+    :param project_data:
+        The file path from which to read data.
+    :return:
+        The YAML data as python objects.
+    """
+    with open(project_data, 'r') as stream:
+        return yaml.load(stream)
+
+
+def append_to_contents(contents, key, values):
+    """
+    Appends data to a dict, adding the recieved values
```

Comment:
recieved -> received ,

and this puppy should be in utilities somewhere

Reply:
actually its specific to this file as it adds to this special key on line 20